### DISCUSSION TITLE
E-Mail Module: Threaded sorting for mails? 2-Column view?

### DISCUSSION THREAD

**Discussion by rostwald** [Intermesh/groupoffice#1115] [Q&A]
Hello everyone,

We have started evaluating GroupOffice as a replacement for our ageing (dead...) Horde installation. So far we are quite pleased with it overall, but the E-Mail Module has some drawbacks which I hope might only be some (hidden?) options I was missing:

Most importantly: is there a threaded sorting for the message list?
Viewing every mail separately and not having the full conversation history is tedious and horribly inefficient, so this is pretty much a must-have for us.

The default 3-column view is rather crowded and not very overseeable, especially on smaller screens.
- Is there a 'classic' 2-Column view with an horizontally split right pane for the messagelist and -preview?
- Can the message list be viewed as a continuous list instead of being separated into short pages that have to be slowly navigated? Especially without proper threaded sorting this is extremely cumbersome...
- Can the message list show the "From" and "Subject" fields as separate columns? Having them cobbled together makes it much harder to (visually) find something (plus, naming that column "from" is simply wrong and misleading if it also contains the subject)

- Is it possible to search (only) in the message body? Or is this only possible via "all fields"?

- Can one set default special folders globally? I.e. our mailserver uses 'INBOX/.Junk', not 'INBOX/.Spam'

There are a few other implementation/configuration details I also might have to ask for help, but as working with email is by far the most important use case for us, this has to fit our needs first before we might proceed.

Thanks for any help!

**Comment by slu-010101** [comment]
Point two is also interesting for us, but there was unfortunately no feedback to our questions:
https://github.com/Intermesh/groupoffice/discussions/1111
https://github.com/Intermesh/groupoffice/discussions/1109
